test/WebJobs.Script.Tests.Integration/Host/StandbyManager/StandbyManagerE2ETests_Windows.cs

```diff
@@ -25,14 +27,16 @@ namespace Microsoft.Azure.WebJobs.Script.Tests
 {
     [Trait(TestTraits.Category, TestTraits.EndToEnd)]
     [Trait(TestTraits.Group, TestTraits.StandbyModeTestsWindows)]
-    public class StandbyManagerE2ETests_Windows : StandbyManagerE2ETestBase
+    public class StandbyManagerE2ETests_Windows : StandbyManagerE2ETestBase, IClassFixture<AzuriteFixture>
     {
+        private readonly AzuriteFixture _azurite;
         private static IDictionary<string, string> _settings;
 
-        public StandbyManagerE2ETests_Windows()
+        public StandbyManagerE2ETests_Windows(AzuriteFixture azurite)
         {
             Utility.ColdStartDelayMS = 1000;
 
+            _azurite = azurite;
```

Comment:
Switch line 67 to `await File.WriteAllTextAsync` ?

Reply:
Not to concerned about that in a test.